components/src/lib/Modal.stories.tsx

```
@@ -0,0 +1,183 @@
+    }
+    return (
+      <>
+        <button onClick={onClose}>열림버튼</button>
+        <Modal {...args} onClose={onClose}>
+          <Modal.ModalContainer
+            isOpen={isOpen}
+            position={args.position}
+            width={args.width}
+          >
+            <Modal.ModalCloseButton onClose={onClose} />
+            <Modal.ModalTitle children={args.children}></Modal.ModalTitle>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)